Source/Applications/SystemCenterNotification/Scripts/TSX/EmailTypes/ByEmailType.tsx

```diff
@@ -141,9 +142,11 @@ const ByEmailType = (props: IProps) => {
                             <fieldset className="border" style={{ padding: '10px', height: '100%' }}>
                                 <legend className="w-auto" style={{ fontSize: 'large' }}>Actions:</legend>
                                 <form>
```

Comment:
do we really need a `form`? Seems like a `div` would be better suited with the appropriate class to remove the `<br />`